django/contrib/admin/helpers.py

```diff
@@ -225,44 +225,53 @@ class InlineAdminFormSet:
     A wrapper around an inline formset for use in the admin system.
     """
     def __init__(self, inline, formset, fieldsets, prepopulated_fields=None,
-                 readonly_fields=None, model_admin=None):
+                 readonly_fields=None, uneditable_fields=None, has_add_permission=True,
+                 has_change_permission=True, has_delete_permission=True, model_admin=None):
```

Comment:
Ok I'll add had_view_permission as well and put the news parameters at the end. I think has_*_permissions are more explicit than flags.